analyzers/IBMXForce/IBMXForce_Lookup.json

```diff
@@ -37,6 +37,13 @@
       "required": true,
       "multi": false,
       "type": "string"
+    },
+    {
+      "name": "verify",
+      "description": "Enable/Disable certificate verification",
+      "required": false,
+      "multi": false,
+      "type": "string"
```

Comment:
This can be ob type `boolean`, instead of making it a `string` and convert it to `boolean`

Reply:
Could be 
```
{
      "name": "verify",
      "description": "Enable/Disable certificate verification",
      "type": "boolean",
      "required": false,
      "multi": false,
      "default": true
    }
```

Reply:
I'll fix this change

Reply:
Hi Nabil, sorry I thought you already adjusted the suggestion. I did not know there was a boolean option (could not find it for some reason, probably my fault) If this is changed, then we can remove the convert to boolean part as well

```
    def str2bool(self, v):
         if isinstance(v, str) or isinstance(v, int) or isinstance(v, unicode) :
             return v.lower() in ("yes", "true", "t", "1")
         if isinstance(v, bool):
             return v
```
If you are busy, I can try to fix it today? 😄

Reply:
@gekkeharry13 oh don't worry it was not a big deal ;) thanks for the contribution